src/ResourceManagement/Authorization/Microsoft.Azure.Management.Authorization/project.json

```diff
@@ -20,36 +20,36 @@
   },
 
   "dependencies": {
+    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.5,4.0.0)"
   },
 
   "frameworks": {
     "net45": {
+      "buildOptions": {
+        "define": [ "NET45" ],
+        "debugType": "portable"
+      },
       "dependencies": {
-        "Microsoft.Rest.ClientRuntime.Azure": "[3.1.0,4.0.0)"
       }
     },
     "netstandard1.5": {
-      "imports": ["dnxcore50"],
+      "buildOptions": {
+        "define": [ "NETSTANDARD15", "PORTABLE" ],
```

Comment:
@hovsepm if we are moving to Net Standard 1.6, do we still need net standard 1.5 monikers?